src/creative.js

```diff
@@ -27,20 +27,28 @@ const DEFAULT_CACHE_PATH = '/pbc/v1/cache';
  */
 
 /**
+ * Public render ad function to be used in dfp creative setup
  * @param  {object} doc
  * @param  {string} adId
  * @param  {dataObject} dataObject
  */
 pbjs.renderAd = function(doc, adId, dataObject) {
-  if (environment.isAmp(dataObject)) {
-    renderAmpAd(dataObject.cacheHost, dataObject.cachePath, dataObject.uuid);
+  if(environment.isMobileApp(dataObject)) {
+    renderAmpOrMobileAd(dataObject.cacheHost, dataObject.cachePath, dataObject.uuid, true);
```

Comment:
why pass a flag for isMobile? Just create 2 functions.

Reply:
Lot of code is common between these two functions so decided to added a flag